dali/python/nvidia/dali/ops.py

```diff
@@ -43,6 +43,7 @@ def _docstring_generator(cls):
 
     schema = b.GetSchema(op_name)
     ret = pre_doc
+    ret += '.. _' + op_name + ':\n'
```

Comment:
Operators already have reference links, as you can get a direct link, for example: https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/supported_ops.html#nvidia.dali.ops.BBoxPaste
No idea how to get it in python though.